services/plugin/src/Plugin.tsx

```diff
@@ -32,17 +32,12 @@ export const Plugin = ({
     pluginSource?: string
     pluginShortName?: string
     propsToPass: any
+    height?: number
+    width?: number
 }): JSX.Element => {
     const iframeRef = useRef<HTMLIFrameElement>(null)
 
-    // we do not know what is being sent in passed props, so for stable reference, memoize using JSON representation
-    const propsToPassNonMemoizedJSON = JSON.stringify(propsToPassNonMemoized)
-    const propsToPass: any = useMemo(
```

Comment:
I removed this as I think it was unnecessary extra memoization, though it's not strictly related to the goal of the ticket.